addons/point_of_sale/models/pos_config.py

```diff
@@ -335,6 +335,10 @@ def _check_company_journal(self):
         if self.journal_id and self.journal_id.company_id.id != self.company_id.id:
             raise ValidationError(_("The sales journal and the point of sale must belong to the same company."))
 
+    def _check_country(self):
```

Comment:
The check in the french localization is enough. In my opinion, we can skip doing this check in stable. I'll write a task to check whether it's really necessary to add this restriction in master.

Reply:
@caburj Indeed, forgot to remove it from the PR...
Fixed!